optuna/integration/wandb.py

```diff
@@ -72,6 +122,9 @@ def objective(trial):
             Set of arguments passed when initializing Weights & Biases run.
             Please refer to `Weights & Biases API documentation
             <https://docs.wandb.ai/ref/python/init>`_ for more details.
+        as_multirun:
+            Creates new runs for each trial. Useful for generating W&B Sweeps like
+            panels (for ex., parameter importance, parallel coordinates, etc).
```

Comment:
Note - At the moment wandb attempts to upload data after every trial in this mode, which in some cases visibly affects performance. Possible solution is to cache all runs locally and then upload once when optimization is finished, but that might require introducing additional callback hooks in Optuna. Leaving for future work.